packages/v4/patternfly-docs/content/design-guidelines/components/input-group/input-group.md

```diff
@@ -7,6 +7,6 @@ section: components
 
 Input groups can combine multiple elements. For example you could add buttons to 1 or both sides of a text input, like in a search field or an email input field. 
 
-<img src="./img/input-group.png" alt="Example of input groups" width="496"/>
+<img src="./img/input-group.png" alt="Example of input groups" width="506"/>
```

Comment:
So actually, we should remove the search example in here, and just have the bottom two, because we're no longer using input-groups to create search inputs!! So basically just use the same image but delete the top example with the arrow :)